docker/compose.testing.yaml

```
@@ -105,13 +100,4 @@ services:
-    restart: unless-stopped
-    # https://docs.docker.com/reference/compose-file/services/#volumes
-    volumes:
-      - source: postgres_test_data
-        target: /var/lib/postgresql/data
-        type: volume
-# https://docs.docker.com/reference/compose-file/volumes/
-volumes:
-  minio_test_data:
-  postgres_test_data:
\ No newline at end of file
+    restart: unless-stopped
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)